src/Stan/Observation.hs

```diff
@@ -13,7 +13,7 @@ module Stan.Observation
     , prettyShowObservation
     ) where
 
-import GHC (SrcSpan)
+import SrcLoc (RealSrcSpan)
```

Comment:
I had to change this since it's how it's stored in HIE files, but fortunately, it still has all necessary information